README.md

```diff
@@ -1,34 +1,115 @@
+#### Multi-user support
+
 - Google authenticator for secure authentication during login.
 - Users can create multiple trips, invite others to collaborate
 - Real-time sync ensures seamless collaboration and updates when multiple users work on the same trip.
 - Participants can edit trip details / leave or delete the trip. They can also add photos that are shared with each other
 
-### Social Aspect
+#### Social Aspect
+
 - Users can decide to make the trip public, so other people can discover it via their feed page.
 - Friend requests and trip invites ensure that the users are not added to trips by bad actors / spammers.
 
-### Other features
+#### Other features
+
 - Users can search for their acquaintances and add them as "friends"
 - They can also search for places and activities and view relevant information such as opening hours, address and photos
 - They can add interests in the profile page that will be taken into account by the AI assistant when generating trip and activity suggestions
 
-### Sensor use
+#### Sensor use
+
 - GPS is used to find nearby restaurants, museums, and activities based on the user's current location.
 - The camera is used to change the trip's thumbnail, change profile picture and add shared pictures.
-- 
-### Offline mode
+-
+
+#### Offline mode
+
 - Previously planned schedules are still available offline, allowing for uninterrupted access even in areas with poor connectivity.
 - The search for users and the feed still work even without connexion, though a bit slower
 - When the user goes offline a red banner will be displayed until they reconnect and certain functions are disabled
-- Photos for users and trips are cached so available offline.
\ No newline at end of file
+- Photos for users and trips are cached so available offline.
+
+## Technical Summary:
+
+#### Tech Stack:
+
+- Frontent: JetpackCompose
+- Backend: Firestore
+- Auth: Firebase Auth
+- Sotrage: Firebase Cloud Storage
+- Other APIs: Google Maps and Places API (for location search and displaying maps), Gemini (for JSON schema generation)
+- CI/CD: Github Actions, SonarCloud (for code quality)
+
+#### Architecture:
+
+Architrecure type: MVVM; Diagram visible [here](https://github.com/swent-t16/Voyageur/blob/main/resources/architecture_diagram.png)
+
+##### View Models:
+
+- UserViewModel:
+
+  - ViewModel responsible for managing user data and interactions with Firebase authentication and user repository.
+  - It provides functionality to load, update, and observe user data, and to handle user-related actions such as signing in / out, adding/removing contacts, and searching users and listening for real time friend request updates.
+
+- TripViewModel:
+
+  - ViewModel for managing trip-related data and operations in the app.
+
+  - This ViewModel interacts with the `TripRepository` to fetch, create, update, and delete trips. It
+  - Handles the management of trip-related states such as the selected trip, selected day, activities, and UI state.
```

Comment:
I think this should be on the above line?

Reply:
I will remove the "`It`" on the line above